test/crypto.ts

```diff
@@ -50,10 +50,18 @@ test('Get', async() => {
 
 test('Get raw ', async() => {
     user = await User.get({id: user.id}, {hidden: true, parse: false})
-    expect(user.pk.S).toMatch(/^User#/)
-    expect(user.email.S).toBeDefined()
-    expect(user.email.S).not.toMatch('peter@example.com')
-    expect(user.email.S).toMatch(/^primary/)
+    if (isV3()) {
```

Comment:
User.get should not ever be returning email.S.  Should these values not be unmarshalled into just a pure email?

Reply:
as explained here https://github.com/sensedeep/dynamodb-onetable/pull/270#issuecomment-1030811573 it is because this test asks for `parsed: false` values